privacy-policy.md

```diff
@@ -0,0 +1,34 @@
+# Privacy Notice
```

Comment:
do we need this in the root directory for FB? or can we put it in some `fb-messenger-bot` directory at root?